odom_transformer:
  ros__parameters:
    base_frame: odom_base
    parent_frame: drone0/map

    odoms_to_transform: ["mid360"]

    mid360:
      topic_in: /Odometry
      source_frame: body
      topic_out: /drone0/odom_base


